### DISCUSSION TITLE
Question about the Quit menu

### DISCUSSION THREAD

**Discussion by halsten-dev** [fyne-io/fyne#4797] [Q&A]
Hi, 

I started using the menu and it's working fine. It's a bit sad to be forced to register the shortcut on the canvas too to make the shortcuts work, but it works so that's ok.

The question is about the Quit menu item that is automatically added. Sadly it seems not very possible to modify it to add a shortcut or even making my own do not work. When I call ```app.Quit()``` from a shortcut I get this kind of error : 

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x58 pc=0xa09b7c]

Does anyone have achieved something like that ?

Thanks in advance.

**Comment by andydotxyz** [comment]
Quit shortcuts should be handled by the operating system.

Calling App.Quit should not panic though, sounds like maybe you should open a bug to report this.

(Though a nil error more likely means you don't actually have access to the app when you called Quit, but on a nil variable instead?)

**Reply by halsten-dev** [reply]
I don't know but I had just a call to app.Quit on a menu.

But on linux many apps quits on Ctrl+Q, like on MacOS. But by default it don't do that with Fyne apps. So I assumed I had to manage it myself. What do you think ?

**Reply by andydotxyz** [reply]
Please open a bug with the full stack trace to allow us to look into this further.
